Views/Shared/_Layout.cshtml

```diff
@@ -6,37 +6,42 @@
     <title>@ViewData["Title"] - Team Modernyze</title>
     <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
     <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
+     <link rel="stylesheet" href="~/css/bootstrap-unlit.css" asp-append-version="true" />
     <link rel="stylesheet" href="~/ModernyzeWebsite.styles.css" asp-append-version="true" />
 </head>
 <body>
 @using Microsoft.AspNetCore.Http;
-<header>
-    <nav class="navbar navbar-expand-sm navbar-toggleable-sm navbar-dark bg-dark border-bottom box-shadow mb-3">
-        <div class="container-fluid">
-            <a class="navbar-brand text-light" asp-area="" asp-controller="Home" asp-action="Index">Team Modernyze</a>
-            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target=".navbar-collapse" aria-controls="navbarSupportedContent"
-                    aria-expanded="false" aria-label="Toggle navigation">
-                <span class="navbar-toggler-icon"></span>
-            </button>
-            <div class="navbar-collapse collapse d-sm-inline-flex justify-content-between">
-                <ul class="navbar-nav flex-grow-1">
+    <header>
+        <meta name="color-scheme" content="dark light">
+        <nav class="navbar navbar-expand-sm navbar-toggleable-sm navbar-dark bg-dark border-bottom box-shadow mb-3">
+            <div class="container-fluid">
+                <a class="navbar-brand text-light" asp-area="" asp-controller="Home" asp-action="Index">Team Modernyze</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target=".navbar-collapse" aria-controls="navbarSupportedContent"
+                        aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="navbar-collapse collapse d-sm-inline-flex justify-content-between">
+                    <ul class="navbar-nav flex-grow-1">
 
-                    <!-- Place any admin-restriced items here -->
-                    @if (!string.IsNullOrEmpty(Context.Session.GetString("HighestPermission"))) {
+                        <!-- Place any admin-restriced items here -->
+                        @if (!string.IsNullOrEmpty(Context.Session.GetString("HighestPermission"))) {
                         if (Context.Session.GetString("HighestPermission").Equals("Administrator")) {
-                            <li class="nav-item">
-                                <a class="nav-link text-light" asp-area="" asp-controller="User" asp-action="Admin">Admin Panel</a>
-                            </li>
+                        <li class="nav-item">
+                            <a class="nav-link text-light" asp-area="" asp-controller="User" asp-action="Admin">Admin Panel</a>
```

Comment:
Same suggestion for text color as written previously. Repeat for any other `text-light` occurrences.